src/plugins/intel_cpu/src/emitters/snippets/aarch64/jit_loop_emitters.cpp

```diff
@@ -0,0 +1,266 @@
+    return begin_expr;
+}
+
+jit_loop_end_static_emitter::jit_loop_end_static_emitter(dnnl::impl::cpu::aarch64::jit_generator* h, dnnl::impl::cpu::aarch64::cpu_isa_t isa,
+                                                         const ov::snippets::lowered::ExpressionPtr& expr)
+    : jit_loop_end_emitter(h, isa, expr) {
+    const auto loop_end = ov::as_type_ptr<snippets::op::LoopEndStatic>(expr->get_node());
+    OV_CPU_JIT_EMITTER_ASSERT(loop_end != nullptr, "Expected LoopEndStatic expr");
+    work_amount = static_cast<int64_t>(loop_end->get_work_amount());
+    is_incremented = loop_end->get_is_incremented();
+    ptr_increments = loop_end->get_ptr_increments();
+    finalization_offsets = loop_end->get_finalization_offsets();
+    data_sizes = loop_end->get_element_type_sizes();
+    evaluate_once = loop_end->get_evaluate_once();
+}
+
+void jit_loop_end_static_emitter::validate_arguments(const std::vector<size_t> &in, const std::vector<size_t> &out) const {
+    const auto io_size = num_inputs + num_outputs;
+    OV_CPU_JIT_EMITTER_ASSERT(out.size() == 0, "Invalid number of out arguments: expected ", 0, " got ", out.size());
+    OV_CPU_JIT_EMITTER_ASSERT(in.size() == io_size + 1, "Invalid number of in arguments: expected ", io_size + 1, " got ", in.size());
+    OV_CPU_JIT_EMITTER_ASSERT(ptr_increments.size() == io_size, "Invalid ptr_increments size: expected ", io_size, " got ", ptr_increments.size());
+    OV_CPU_JIT_EMITTER_ASSERT(finalization_offsets.size() == io_size,
+                              "Invalid finalization_offsets size: expected: ", io_size, " got ", finalization_offsets.size());
+    OV_CPU_JIT_EMITTER_ASSERT(data_sizes.size() == io_size, "Invalid data_sizes size: expected: ", io_size, " got ", data_sizes.size());
+}
+
+void jit_loop_end_static_emitter::emit_code(const std::vector<size_t> &in, const std::vector<size_t> &out,
+                                            const std::vector<size_t> &pool_vec_idxs, const std::vector<size_t> &pool_gpr_idxs) const {
+    validate_arguments(in, out);
+    emit_impl(in, out);
+}
+
+void jit_loop_end_static_emitter::emit_impl(const std::vector<size_t>& in, const std::vector<size_t>& out) const {
+    std::vector<size_t> data_ptr_reg_idxs;
+    data_ptr_reg_idxs.reserve(num_inputs + num_outputs);
+    std::copy(in.begin(), in.end() - 1, std::back_inserter(data_ptr_reg_idxs));
+
+    XReg reg_work_amount = XReg(in.back());
+    if (!evaluate_once) {
+        for (size_t idx = 0; idx < data_ptr_reg_idxs.size(); idx++) {
+            if (!is_incremented[idx] || ptr_increments[idx] == 0)
+                continue;
+            XReg data_reg = XReg(data_ptr_reg_idxs[idx]);
+            if (ptr_increments[idx] > 0) {
+                h->add(data_reg, data_reg, ptr_increments[idx] * wa_increment * data_sizes[idx]);
+            } else if (ptr_increments[idx] < 0) {
+                h->sub(data_reg, data_reg, - ptr_increments[idx] * wa_increment * data_sizes[idx]);
+            }
```

Comment:
Applied for finalization offset. Yes, h->add on aarch64 accepts only uint32, so a negative number will be accepted as a large positive offset. I've checked L243 and L255. It's OK, because the offset are stored in XReg, no such issues. BTW, if a number in XReg must be compared in runtime, I usually use cmp-jmp instructions. Thanks Alexandra!